build.gradle.kts

```
@@ -4,9 +4,12 @@ plugins {
 }
 
 val swagger: String by project
-val rsApi : String by project
+val rsApi: String by project
 
 allprojects {
+    version = "1.0.0-SNAPSHOT"
+    group = "org.eclipse.dataspaceconnector"
```

```suggestion
    group = "org.eclipse.dataspaceconnector.registrationservice"
```

to make the repo distinction clear